opm/input/eclipse/Schedule/Network/Branch.hpp

```diff
@@ -45,6 +45,7 @@ class Branch {
 
     const std::string& downtree_node() const;
     const std::string& uptree_node() const;
+    void set_uptree_node(const std::string& new_uptree_node) { m_uptree_node = new_uptree_node; }
```

Comment:
The implementation of the function better be in the cpp file.

Reply:
ok!